tools/container/toolkit/toolkit.go

```diff
@@ -436,6 +451,24 @@ func installToolkitConfig(c *cli.Context, toolkitConfigPath string, nvidiaContai
 	// Use the driver run root as the root:
 	driverLdconfigPath := config.NormalizeLDConfigPath("@" + filepath.Join(opts.DriverRoot, strings.TrimPrefix(ldconfigPath, "@/")))
 
+	var ctrRuntimeBinPaths []string
+	ctrRuntimeBinPaths, err = getRuntimeBinaryPaths(runtime, hostRoot)
+	if err != nil {
+		log.Warningf("Error retrieving runtime binary paths: %v", err)
+	}
+
+	var ctkRuntimes []string
+	defaultCfg, err := cfg.Config()
```

Comment:
Note that strictly speaking since we're loading the config source, this is not the default config, but the config from the file in the container.